livekit-protocol/src/livekit.rs

```diff
@@ -78,6 +78,14 @@ pub struct EventMetric {
     #[prost(uint32, tag="9")]
     pub rid: u32,
 }
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MetricsRecordingHeader {
+    #[prost(string, tag="1")]
+    pub room_id: ::prost::alloc::string::String,
+    #[prost(bool, optional, tag="2")]
+    pub enable_user_data_training: ::core::option::Option<bool>,
+}
 //
 // Protocol used to record metrics for a specific session.
```

Comment:
I had to update the `livekit-protocol` package to the latest in order to get access to `TokenSourceRequest` / `TokenSourceResponse` from [here](https://github.com/livekit/protocol/pull/1224) - that's where the majority of the diff changes are coming from:

<img width="168" height="65" alt="Image" src="https://github.com/user-attachments/assets/27de9788-c6b8-494e-af15-318f6f3904d1" />

Is this something worth doing as a separate pull request merged in as a prerequisite or is it fine to include this broader change in here?

Reply:
I followed the instructions in [here](https://github.com/livekit/rust-sdks/blob/main/livekit-protocol/generate_proto.sh#L17) so I think I did generate these with the correct version, FYI. That being said I also added a note for myself to add some better documentation (maybe add it to the `README.md`?) on how to generate these, it took me longer than it should have for me to self-discover how.